readme.md

```diff
@@ -1088,7 +1088,7 @@ Parses the arguments and runs the app.
 
 ##### argv
 
-Type: `Array`\
```

Comment:
These backslashes are necessary for "Default" text to be on the next line after "Type".

Reply:
Whoops, doesn't know about this. Return them back!